### DISCUSSION TITLE
Consultation period extended to April 3rd 2024

### DISCUSSION THREAD

**Discussion by MFSCSamuel** [MinistryFinancialServices/Virtual-Asset-Amendment-Consultation#31] [Announcements]
Dear users,

Based on the feedback received, the Ministry is happy to announce a 14-day extension to the ongoing consultation on proposed amendments to the Virtual Asset (Service Providers) Act. We will now be accepting comments up to April 3rd, 2024.

In addition to this platform, feedback will also be considered via email responses to dfslegislation@gov.ky.

**Comment by Aaron-XReg** [comment]
A big thank you to the Ministry! Let's use the time to collaborate on Github and help shape Cayman's virtual asset landscape.

**Comment by yaksetig** [comment]
Thank you very much for the extension!

**Comment by kennethng-ledn** [comment]
Thank you for accommodating the extension request, we very much appreciate this opportunity to contribute to this consultation process.

**Comment by NotableVentures** [comment]
Thank you!
